workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml

```
@@ -0,0 +1,37 @@
+extension: workflow
+products: ["insightconnect"]
+name: IVM_Top_Remediations_to_JIRA 
```

```suggestion
name: InsightVM_Top_Remediations_to_JIRA 
```

And move to updated directory to reflect too.